pkgs/games/prismlauncher/wrapper.nix

```diff
@@ -21,10 +25,16 @@
 , msaClientID ? null
 , gamemodeSupport ? stdenv.isLinux
 , textToSpeechSupport ? stdenv.isLinux
+, waylandSupport ? false
```

Comment:
a small comment explaining what this does could be good

Reply:
Added, with a warning.